src/modules/health/indicators/dynamic-config.health.ts

```
@@ -0,0 +1,50 @@
+        }),
+      );
+    } catch (error) {
+      throw new HealthCheckError(
+        'DynamicConfig health check failed',
+        this.getStatus(key, false, {
+          error: error instanceof Error ? error.message : String(error),
+        }),
+      );
+    }
+  }
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)